akka-remote/src/main/java/akka/remote/artery/AeronErrorLog.java

```diff
@@ -39,24 +36,25 @@
  */
 public class AeronErrorLog
 {
-    private final File cncFile;
     final MappedByteBuffer cncByteBuffer;
-    final DirectBuffer cncMetaDataBuffer;
-    final int cncVersion;
     final AtomicBuffer buffer;
 
     public AeronErrorLog(File cncFile)
     {
-      this.cncFile = cncFile;
+      this(cncFile, NoLogging.getInstance());
```

Comment:
Perhaps mark this constructor as deprecated so people are less likely to accidentally have no logging here?

Reply:
This seems like a good comment @guidomedina

Reply:
And you are right, I had added it, but I'm travelling in Cyprus and lost that minor change because I didn't pull my own remote branch, fixing...thanks for pointing it out.

Reply:
@ktoso that's now fixed, sorry about that...